sig-scheduling/charter.md

```diff
@@ -0,0 +1,65 @@
+# SIG Scheduling Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+SIG Scheduling is responsible for the components that make Pod placement decisions.
+We build Kubernetes schedulers and scheduling features for Pods. We design and
+implement features that allows users to customize placement of Pods on the nodes
+of a cluster. These features include those that improve reliability of workloads,
+more efficient use of cluster resources, and/or enforces placement policies. 
+
+### In scope
+
+Link to SIG section in [sigs.yaml]
+
+#### Code, Binaries and Services
+
+- Scheduling related features (e.g. Node Affinity)
+- Kube-scheduler performance and scalability (with [sig-scalability](../sig-scalability))
+- Kube-scheduler reliability (problem detection and remediation)
+- Pod scheduling APIs (with [sig-api-machinery](../sig-api-machinery))
+- Node resource management (with [sig-node](../sig-node))
+- Cluster resource management (with [wg-resource-management](../wg-resource-management))
```

Comment:
how do we define "resource management" ? how do we define "Quota"'s feature, "resource plan"?

Reply:
Good point. I guess we should clarify that we don't work on the enforcement of resource quota.

Reply:
I added an item to the "out of scope" section to clarify. PTAL.

Reply:
LGTM :)

Reply:
Working groups are ephemeral.

Since the working group today is sponsored by sig-node and sig-scheduling, I would just update this to with sig-node.

Reply:
is this line intended to specifically describe the quota api or something broader?

Reply:
This is about cluster level resources, such as dynamic PV, TPU, etc.